sdk/python/kfp/compiler/pipeline_spec_builder.py

```diff
@@ -690,12 +704,25 @@ def build_component_spec_for_group(
                         channel.channel_type))
             component_spec.input_definitions.artifacts[
                 input_name].is_artifact_list = channel.is_artifact_list
-        else:
-            # channel is one of PipelineParameterChannel, LoopArgument, or
-            # LoopArgumentVariable.
+        elif isinstance(channel, for_loop.LoopArtifactArgument):
```

Comment:
Can we bundle this into the condition above, consistent with how we handle parameters below?

`if isinstance(channel, (pipeline_channel.PipelineArtifactChannel, for_loop.LoopArtifactArgument)):`

Reply:
Done.